package.json

```diff
@@ -23,19 +24,24 @@
   "devDependencies": {
     "babel-register": "^6.26.0",
     "eslint": "^6.8.0",
+    "eslint-plugin-testcafe": "^0.2.1",
     "got": "^10.4.0",
     "gulp": "^4.0.2",
     "gulp-eslint": "^6.0.0",
     "gulp-run": "^1.7.1",
     "mocha": "^7.0.1",
-    "supertest": "^4.0.2"
+    "supertest": "^4.0.2",
+    "testcafe": "^1.9.3"
   },
   "dependencies": {
-    "express": "^4.17.1",
-    "express-fileupload": "^1.2.0",
+    "@ffmpeg-installer/ffmpeg": "^1.0.20",
```

Comment:
Let's remove 
`@ffmpeg-installer/ffmpeg`
`testcafe-browser-tools`
since we don't need them in our project